Makefile.dep

```diff
@@ -606,6 +606,14 @@ ifneq (,$(filter random,$(USEMODULE)))
   endif
 endif
 
+ifneq (,$(filter random_seed,$(USEMODULE)))
+  USEMODULE += luid
+  USEMODULE += xtimer
+
+  FEATURES_REQUIRED += periph_hwrng
+  FEATURES_REQUIRED += periph_adc
```

Comment:
@kaspar030 `FEATURES_PROVIDED` is not set at that point in time. I re-checked it and we already had this problem in #6852. Am I blind to see your proposals? Can you give an example of what works?